NitroxClient/MonoBehaviours/Gui/HUD/RemotePlayerVitals.cs

```diff
@@ -132,6 +136,22 @@ private void CreatePlayerNameText()
             playerNameText.layer = em.gameObject.layer;
             // em does not have a parent anymore on latest stable, so the stats position is incorrect.
             playerNameText.transform.parent = em.transform;//.parent.transform;
+            GameObject signalBase = Instantiate(Resources.Load("VFX/xSignal")) as GameObject;
+            signalBase.name = "signal " + playerName;
+            signalBase.transform.position = Player.main.transform.position;
+            
+            PingInstance ping = signalBase.GetComponent<PingInstance>();
+            ping.visible = false;
+            ping.name = playerName;
+
+            ping.pingType = PingType.None;
+            ping.displayPingInManager = true;
+            ping.maxDist = 0.01f;
+            ping.minDist = 0f;
+
+            pingId = PingManager.Register(ping);
```

Comment:
I assume this works on stable, but the function returns nothing (`void`) on experimental - this will stop working soon.

Reply:
no i changed the function so it returns the int ID

Reply:
That's not a good idea, we want to change the least game code possible, and inject everything else at runtime. Currently we're only adding a single function call to the game to start everything up, and we want to keep it at just that.

Consider finding a different way, simpelest and most straightforward would be store the `PingInstance` as a private field in the class.